### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by gewfy**
- [x] New onboarding
- [x] Add local video and image assets
- [x] Fallback if real sessions are not fetched
- [x] Remove old welcome/early access
- [x] Skip instead of continue

| Welcome | Page 1 | Page 2 | 
|-|-|-|
| image image image | image image image | image image image |

| Page 3 | Page 4 |
|-|-|
| image image image | image image image |

https://user-images.githubusercontent.com/474066/231679160-a8cb7686-f8b6-45c4-a4d9-f4391419df17.mp4

### CODE REVIEW

File: client/src/routes/screens/EarlyAccessInfo/EarlyAccessInfo.tsx
Comment (transient-jonas) [inline client/src/routes/screens/EarlyAccessInfo/EarlyAccessInfo.tsx:1]: 🔥

File: client/src/routes/screens/Onboarding/components/AboutCarousel.tsx
Comment (transient-jonas) [inline client/src/routes/screens/Onboarding/components/AboutCarousel.tsx:1]: Neat!!! 💥 🤘

File: client/src/routes/screens/Onboarding/components/Page2.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            />
+            <NameWrapper>
+              <Name>{t('page2.name1')}</Name>
+            </NameWrapper>
+          </Frame>
+        </Square1>
+        <Square2>
+          <Frame>
+            <Video
+              sources={[
```
Comment (swemail) [inline client/src/routes/screens/Onboarding/components/Page2.tsx:100]: and here

File: client/src/routes/screens/Onboarding/components/Page2.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            />
+            <NameWrapper>
+              <Name>{t('page2.name2')}</Name>
+            </NameWrapper>
+          </Frame>
+        </Square2>
+        <Square3>
+          <Frame>
+            <Video
+              sources={[
```
Comment (swemail) [inline client/src/routes/screens/Onboarding/components/Page2.tsx:115]: here too

File: client/src/routes/screens/Onboarding/components/Page2.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+const Page2 = () => {
+  const {t} = useTranslation('Screen.Onboarding');
+
+  return (
+    <>
+      <Squares>
+        <Square1>
+          <Frame>
+            <Video
+              sources={[
```
[the remark this answers is not in the document]
Comment (gewfy) [reply client/src/routes/screens/Onboarding/components/Page2.tsx:85]: Doh ofc! Thanks

**Review by swemail — APPROVED** [review]
Grrrrrreat!

File: client/src/routes/screens/Onboarding/components/Page1.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+};
+
+const Page1 = () => {
+  const {t} = useTranslation('Screen.Onboarding');
+  const opacity = useSharedValue(0);
+  const [sessions, setSessions] = useState<LiveSession[]>(
+    getFallbackSessions(),
+  );
+
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (gewfy) [reply client/src/routes/screens/Onboarding/components/Page1.tsx:81]: Yeah the shared `opacity` object never changes. But it's value `opacity.value` does :)

**Review by transient-jonas — APPROVED** [review]

**Pull request merged**